README.md

```diff
@@ -22,10 +22,17 @@ A data-driven `UICollectionView` framework for building fast and flexible lists.
 
 `IGListKit` is built and maintained by [Instagram engineering](https://engineering.instagram.com/), using the open source version for the Instagram app.
 
+## Requirements
+
+- Xcode 8.0+
+- iOS 8.0+
+- Interoperability with Swift 3.0+
```

Comment:
Ya I think we should call out that we only support 3.0. I think there _are_ problems on 2.3 tho (see #65).